src/main/java/cz/brmlab/yodaqa/flow/dashboard/QuestionAnswer.java

```diff
@@ -11,24 +11,19 @@
 	protected String text;
 	protected double confidence;
 	protected int ID;
-	protected String source;
-	protected List<Integer> passageIDs= new ArrayList<>();
+	protected List<Integer> snippetIDs= new ArrayList<>();
 	public QuestionAnswer(String text, double confidence) {
 		this.text = text;
 		this.confidence = confidence;
 	}
-	public List<Integer> getPassageIDList() {return passageIDs;}
-	public void addToPassageList(int passageID){
-		passageIDs.add(passageID);
+	public List<Integer> getSnippetIDs() {return snippetIDs;}
+	public void addToSnippetIDList(int snippetID){
+		snippetIDs.add(snippetID);
 	}
-	public List<Integer> getPassageList(){
-		return passageIDs;
-	}
-	/** @return the text */
+ 	/** @return the text */
 	public String getText() { return text;}
 	/** @return the confidence */
 	public double getConfidence() { return confidence; }
 	public void setID(int ID) {this.ID = ID;}
```

Comment:
It probably should, but it opens the question whether we need an ID for QuestionAnswers at all, considering we copy the SnippetID's between answers anyway. Atleast, I never noticed this, because we don't print the QuestionAnswer ID anywhere.